javascript/ql/src/Security/CWE-020/IncompleteUrlRegExp.ql

```diff
@@ -0,0 +1,70 @@
+/**
+ * @name Incomplete URL regular expression
+ * @description Security checks on URLs using regular expressions are sometimes vulnerable to bypassing.
```

Comment:
Doesn't seem to follow the standard format of "doing X may cause Y".

Perhaps if the name of the query was more specific to unescaped meta-characters, this could be phrased more directly.